sdk/__init__.py

```diff
@@ -25,7 +25,6 @@
 class DeviceType(StrEnum):
     QPU = "QPU"
     EMU_FREE = "EMU_FREE"
-    EMU_SV = "EMU_SV"
```

Comment:
removing this will break e2e

Reply:
@awennersteen Why don't we use `EMU_FREE` in the e2e tests?

Reply:
Because "EMU_FREE" routes jobs to the Pasqman right now, versus "EMU_SV" routes them to the DGX connector which allows us to test that pathway!